### ISSUE TITLE
Where is the utility to convert projects from package.config to package reference?

### ISSUE THREAD

**Issue by StingyJack** [NuGet/Home#6509]
I saw a writeup for one here somewhere, but haven't seen one materialized yet. The current documentation lists steps that would take ~4.5 weeks (at ~15 minutes per project to update and quick test) to complete with our current project count.  

There isn't any good way to justify losing that many billable or otherwise productive hours. So if the intent is to effectively coerce this upgrade path, some utility will need to be made to perform this update that works with currently supported components  (TFS 2015 and later, VS 2015 + 2017, MSBuild 14 + 15, etc.)

**Comment by rohit21agrawal** [comment]
@StingyJack it's currently in the works. We are currently tracking this work in the PR: https://github.com/NuGet/NuGet.Client/pull/1749 

If you would like to test out the tool on some of your projects, you can build from source and try it out. @karann-msft might be able to tell you more details about trying out the preview bits through a different channel

**Comment by karann-msft** [comment]
@StingyJack reach out to me on karann [at] microsoft [dot] com and I can share it with you.

**Comment by StingyJack** [comment]
@karann-msft - Thanks, but i'm not looking to continue being an unwitting beta tester for MSFT again this year. Last year was quite enough. This tool should have been ready along with PackageRef, or at the very least when you all announced that you were no longer going to support packageCfg.

**Comment by karann-msft** [comment]
Out of curiosity, what did you beta test last year for Microsoft? 
PS: This utility will ship with VS 15.7 Preview 3 this week.

**Comment by StingyJack** [comment]
The key word there is 
> unwitting

I got the same bug-riddled VS everyone else did. For me the time spent initial troubleshooting on problems before I realized they were defects, coming up with workarounds for those, wading through docs labeled ".net" but were actually netcore, and the time spent re-building a few things to bridge intentionally lost or broken functionality, I lost about two months of productivity last calendar year.  

Beta stuff is going to be even less tested than that "production ready" VS was, and I'm not looking for more time to lose. 

Specifically to this topic, I also have the distinct impression that package ref may be a good concept, but the execution is just wrong in too many ways (look for some mentions:stingyjack here and the nuget docs repo for examples), and everyone over there is so deep into the effort, that there is no unbiased re-evaluation of the plan happening. Thank you again for the offer, but the whole package ref implementation is really looking like something I do not want.
